src/codeu/chat/common/ServerInfoTest.java

```diff
@@ -0,0 +1,22 @@
+package codeu.chat.common;
+import static org.junit.Assert.assertEquals;
+import org.junit.Test;
+import codeu.chat.util.Uuid;
+
+import java.io.IOException;
+
+public class ServerInfoTest {
+
+    @Test
+    public void testServerInfo() {
+        ServerInfo server1 = new ServerInfo();
```

Comment:
I'm a little bit confused as to what you mean by this. For the parameter, is this for the constructor?